_posts/2023-02-14-whatsnew-reporting-cli.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "What’s new: OpenSearch Reporting CLI"
+authors:
+- maharup
+- jdbright
+- jadhanir
+- lijshu
+- alicejw
+date: 2023-02-14 00:00:00 -0700
+categories:
+ - technical-post
+ meta_keywords: "Reporting CLI, automate reports"
+ meta_description: "The Reporting CLI tool makes it convenient to generate reports externally without logging into OpenSearch Dashboards."
+---
```

Comment:
Please update the meta with the following: 

**Meta keywords:**  Reporting CLI tool, OpenSearch reporting plugin, automate reports in OpenSearch
**Meta description:**  Learn how the release of the Reporting CLI tool for OpenSearch makes it convenient to generate external reports without logging into OpenSearch Dashboards.